tests/test_test_tools/test_test_cases.py

```diff
@@ -477,3 +478,23 @@ def test_fail_on_first_run(self):
             raise AssertionError("Expected error to trigger rerun!")
         assert self.run_count == 2, "Should only be rerun once!"
         assert not os.path.isfile(file), "File should not exist"
+
+
+class TestBaseContractTestCase(TestCase):
+    """Test case for BaseContractTestCase ABC class."""
+
+    @mock.patch(
+        "aea.test_tools.test_contract.BaseContractTestCase.sign_send_confirm_receipt_multisig_transaction"
+    )
+    def test_sign_send_confirm_receipt_transaction(
+        self, sign_send_confirm_receipt_multisig_transaction_mock
+    ):
+        """Test sign_send_confirm_receipt_multisig_transaction is called for backward compatibility."""
+
+        class ContractTestCase(BaseContractTestCase):
+            pass
+
+        ContractTestCase.sign_send_confirm_receipt_transaction(
+            "tx", "ledger_api", "crypto"
+        )
+        sign_send_confirm_receipt_multisig_transaction_mock.assert_called_once()
```

Comment:
Please write a proper test for this class. We don't want to just fill coverage gaps.

Reply:
Also move this into its own class `.../test_test_tools/test_test_contract.py`

Reply:
@5A11 does it really worth end-to-end test? The only purpose of this method is to pass params to another one which is covered.

Reply:
We do not have any tests for `contract_test_tools`. 
Need to have a test class that tests it, even though it might be already in the coverage report.